### DISCUSSION TITLE
WW3 error in multi-grid

### DISCUSSION THREAD

**Discussion by meenakshisreejith** [NOAA-EMC/WW3#553] [Q&A]
Presently im using CNRM-CM6-1-HR winds for a hindcast study, but the wind files are running perfectly, using ww3_prnc.inp but as i try to run the model, the error comes. 

w3servmd MPI_ABORT, IEXIT=          13
 w3servmd UNIT missing
 w3servmd MSG missing
 w3servmd FILE missing
 w3servmd LINE missing
 w3servmd COMM missing

Can anyone help in this regard?

**Comment by JessicaMeixner-NOAA** [comment]
@meenakshisreejith this type of error is typically caused by incorrect input fields or missing files, for example if your wind.ww3 or mod_def.ww3 file is not consistent with your model code or in not located in your run-directory.

**Reply by meenakshisreejith** [reply]
Thank You so much..it worked. The problem was actually my mod_def file. I created new files and tried and it worked. Thank You :) 💯
